### ISSUE TITLE
Switching scene with exporter opened leads to error

### ISSUE THREAD

**Issue by AurL** [sketchfab/Unity-glTF-Exporter#6]
The gameobject used for export is deleted and not recreated:
```
MissingReferenceException: The object of type 'ExporterScript' has been destroyed but you are still trying to access it.
```

**Comment by AurL** [comment]
Fixes with `exporterGo.hideFlags = HideFlags.HideAndDontSave;`

**Comment by AurL** [comment]
Fixed in 1.0.0. Closing
